.github/workflows/test.yaml

```diff
@@ -381,40 +381,6 @@ jobs:
       with:
         password: ${{ secrets.PYPI_UPLOAD_TOKEN }}
 
-  # Read the Docs has no support for our "twisted-1.2.3" numbering convention
-  # and can't detect which tag is the stable one.
-  # A workaournd is to manually push to a "stable" branch to inform RTD
-  # that this is what we want at the "/en/stable" link.
-  update-stable-branch:
-    permissions:
-      contents: write
-    name: Update stable branch for RTD - on tag
-    runs-on: 'ubuntu-20.04'
-    needs: [release-publish]
-    if: startsWith(github.ref, 'refs/tags/twisted-')
-    steps:
-    - name: Update stable branch - on stable tag
-      env:
-        STABLE_BRANCH: 'stable'
-        STABLE_REF_RE: '.*twisted-[[:digit:]]+\.[[:digit:]]+\.[[:digit:]]+$'
-      run: |
-          if [[ $GITUB_REF =~ $STABLE_REF_RE ]]; then
-            echo "Pushing current tag to stable branch..."
-            git config --global user.name '$GITHUB_ACTOR'
-            git config --global user.email '$GITHUB_ACTOR@users.noreply.github.com'
-
-            echo '::group::git fetch'
-            git fetch origin
-            echo '::endgroup::'
-
-            git log -1
-            git checkout -b $STABLE_BRANCH origin/$STABLE_BRANCH
-            git reset --hard $GITHUB_SHA
-            git push --force
-            echo "Done"
-          else
-            echo "Branch not updated for not stable releases: $GITUB_REF"
-          fi
```

Comment:
I am confused — why isn't this necessary any more?

I don't know RTD intimately, but I don't understand how the other changes could impact this.

Not really a request for changes on the code itself, I don't want a big comment saying why something *isn't* there, but perhaps some missing context in the PR or issue description.

Thanks for these cleanups though, I would love to see less warning noise from GHA.

Reply:
This code doesn't work.

It is supposed to run only for the latest stable release

Most of the time is skipped, like for this PR.

You can see it run for 22.10.0 release.

It was supposed to update this branch https://github.com/twisted/twisted/tree/stable

But you can see that it just doesn't recognize the stable release, and while it is executed, it skips the update.

https://github.com/twisted/twisted/actions/runs/3360096602/jobs/5568908454#step:2:26

It is hard to troubleshoot it with GitHub Actions

For the last releases, I was just manually updating the branch.

And I don't plan to troubleshoot that code... so is best to remove it.

Reply:
@glyph what we can do to make Twisted "RTD" friendly , is that instead of using `twisted-1.2.3.` as the name for release tags, we just use `1.2.3`

In this way, RTD can detect which tag is "latest"

Reply:
Thanks for the review :)

Reply:
Ah, okay. If it's already broken then let's not keep dragging it around.

Reply:
yes... it was broken for the last 2 releases...at least.
And troubleshooting is hard.

I have a ticket to update the release docs... in on my todo... but that is for another weekend.